Automattic/jetpack
# Add WAF to Jetpack Scan feature lists

⚠️ **This PR should not be merged until Jetpack Firewall is released.** ⚠️

This PR adds "Access to latest Firewall rules" as a bullet point to Jetpack Scan's feature lists.





#### Changes proposed in this Pull Request:

* Adds "Access to latest Firewall rules" to the Scan feature list in the Jetpack plugin.
* Adds "Access to latest Firewall rules" to the Scan feature list in My Jetpack.

#### Jetpack product discussion



1201069996155217-as-1202025817736947

#### Does this pull request change what data or activity we track or use?



No

#### Testing instructions:






* Check out this branch and boot up a test site.
* Go to `/wp-admin/admin.php?page=my-jetpack#/add-scan` and validate that "Access to latest Firewall rules" is included in the feature list.
* Go to `/wp-admin/admin.php?page=jetpack#/product/scan` and validate that "Access to latest Firewall rules" is included in the feature list.

#### Screenshots:

<img width="1014" alt="Screen Shot 2022-04-06 at 1 59 09 PM" src="https://user-images.githubusercontent.com/10933065/162060602-c12c3c46-72c6-4084-b90f-2ea250c3dc31.png">
